dosa.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pani Puri</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<nav>
+    <div class="nav-bar">
+        <h1>Chef Curry's Kitchen</h1>
+        <ul class="nav-elements">
+            <a href="index.html">
+            <li>Home</li>
+            </a>
+            <li>About</li>
+        </ul>
```

Comment:
You don't need to use an unordered list here, it would be more appropriate to just use a div with stand-alone anchor (a) elements in there. Particularly, I don't like nesting the list element (li) inside of the anchor element, this is not intuitive and not the standard for how we nest elements in html.